src/pages/home/Home.js

```diff
@@ -0,0 +1,74 @@
+import './Home.css';
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import HeaderContent from "../../components/headerContent/HeaderContent";
+import Tile from "../../components/tile/Tile";
+import Skeleton from "../../components/skeleton/Skeleton";
+
+function Home() {
+
+    const [getResult, setGetResult] = useState();
+
+    useEffect(() => {
+        const source = axios.CancelToken.source();
+
+        async function fetchData() {
+            try {
+                const result = await axios.get(`https://www.reddit.com/hot.json?limit=15`, {cancelToken: source.token});
+                setGetResult(result.data.data.children);
+                console.log(result.data);
+                console.log(result.data.data.children);
+
+            } catch (error) {
+                if (axios.isCancel(error)) {
+                } else {
+                    throw error;
+                }
```

Comment:
Als je niets in je if hebt, dan zou ik ervoor kiezen om er een `!` voor te zetten, zodat je dan de tegenstelling hebt:
```suggestion
                if (!axios.isCancel(error)) {
                    throw error;
                }
```